test/puppetlabs/trapperkeeper/services/authorization/authorization_core_test.clj

```diff
@@ -190,14 +233,4 @@
         (is (= ["com" "bull" "bald"] (:pattern (first acl))))
         (is (= :exact (:qualifier (first acl))))
         (is (= ["com" "eagle" "bald"] (:pattern (second acl))))
-        (is (= :exact (:qualifier (first acl))))))
-    (testing "Given a rule config with query parameters"
-      (is (= {"env" #{"prod" "staging"}}
-             (-> (merge-with merge multi-query-param base-path-auth)
-                 config->rule
-                 :query-params)))
-      (testing "single values are converted to sets"
-        (is (= {"environment" #{"production"}}
-               (-> (merge-with merge single-query-param base-path-auth)
-                   config->rule
-                   :query-params)))))))
```

Comment:
Moved the query-params test up because they were accidentally under the ACL testing block